.tito/custom.py

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (c) 2016 Red Hat, Inc.
+#
+# All rights reserved.
+#
+# Author: Jan Pokorny <jpokorny@redhat.com>
+# (using src/tito/builder/fetch.py from the tito project as a template)
+#
+# This source code is licensed under the GNU Lesser General Public License
+# version 2.1 or later (LGPLv2.1+) WITHOUT ANY WARRANTY.
+#
+
```

Comment:
I dunno if it's more Pythonic.  It's just defining variables that are
generally unregulated, though possible to express some metadata.

Furthermore, generally copyright notice and authorship declaration
are orthogonal; for instance I only use `__author__` in clufter:

https://pagure.io/clufter/blob/8d6d8366032362c24a7bacb3d59fd206c89bee1d/f/__main__.py

As you can say, I assume Red Hat, my employer, as the entity sharing
the copyright on that piece of work, as this is something I am
getting paid for by that company, and the chances it would take
off regardless are not very high.  It's just an implicit liability
here from my POV.

Next, I have to wonder why the template has everything parameterized
except for the actual contact person :-)

Also, top-level comments are not always appropriate, and this is IMHO
one such case: extension to some external framework, even more so
when it's not intended for the distribution tarball and never intended
to be processed by any sort of documentation tool.  Instead, I added
some hints as non-documentation comments which are equally worthy for
manual comprehension.

IMHO suitability > blind adherence.